distribution/tools/java-version-checker/src/main/java/org/opensearch/tools/java_version_checker/JavaVersionChecker.java

```diff
@@ -52,23 +53,15 @@ public static void main(final String[] args) {
         if (args.length != 0) {
             throw new IllegalArgumentException("expected zero arguments but was " + Arrays.toString(args));
         }
-        if (JavaVersion.compare(JavaVersion.CURRENT, JavaVersion.JAVA_8) < 0) {
+        if (Runtime.version().compareTo(Version.parse("11")) < 0) {
```

Comment:
For the record, I actually doubt we even need this check: pre-11 JVM won't accept the bytecode